lib/v2/swpu/router.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function (router) {
+    router.get('/bgw/:code', require('./bgw'));
+    router.get('/dean/:code', require('./dean'));
+    router.get('/scs/:code', require('./scs'));
+    router.get('/dxy/:code', require('./dxy'));
```

Comment:
```suggestion
    router.get('/dean/:code', require('./dean'));
    router.get('/dxy/:code', require('./dxy'));
    router.get('/scs/:code', require('./scs'));
```
Alphabetical order helps maintenance